hartshorn-core/src/main/java/org/dockbox/hartshorn/core/domain/Exceptional.java

```diff
@@ -21,6 +21,8 @@
 import org.dockbox.hartshorn.core.function.CheckedBiFunction;
 import org.dockbox.hartshorn.core.function.CheckedFunction;
 import org.dockbox.hartshorn.core.function.CheckedSupplier;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
```

Comment:
Quick notice, yesterday I finalized the switch from `org.jetbrains.annotations` to `org.checkerframework`, see 0b8bbf9a8a4ee55259542a25a121c891afac2373, so you'll probably want to update these imports :)